Fleury Algorithm/fleury.py

```diff
@@ -0,0 +1,194 @@
+from collections import defaultdict
```

Comment:
Let's add a header mentioning the time and space complexity, just like [here](https://github.com/iuliagroza/Algorithms/blob/acc56e46a5d7e2fcde914bf413c8046d5729cd20/KMP/KMP.py#L1).